source/MainWindow.cpp

```diff
@@ -224,6 +224,12 @@ BMessage(B_ABOUT_REQUESTED));
 	fResourceList->AddStatusView(fStatsBox);
 	fStatsBox->AddChild(fStatsString);
 
+	BMessenger panelMessenger(this);
+
+	fMergePanel = new BFilePanel(B_OPEN_PANEL, &panelMessenger);
+	fMergePanel->SetButtonLabel(B_DEFAULT_BUTTON, "Merge");
+	fMergePanel->SetRefFilter();
```

Comment:
Does this even compile? Anyway, you can take it out.

This is a good place to change the default window title to "ResourceEdit: Merge".